workbox-sw is needed by the service worker and could not be copied over to your new site.

i just run command ```yarn build```,more error info as follow:

...
601ms additional asset processing
2ms chunk asset optimization
 94% asset optimizationMergetJsonsWebpackPlugin compilation started...
MergetJsonsWebpackPlugin compilation completed...                                                                  58ms asset optimization 95% emittingMergetJsonsWebpackPlugin emit starts...
MergetJsonsWebpackPlugin emit completed...
MergetJsonsWebpackPlugin emit starts...
MergetJsonsWebpackPlugin emit completed...
MergetJsonsWebpackPlugin emit starts...
MergetJsonsWebpackPlugin emit completed...
Error: workbox-sw is needed by the service worker and could not be copied over to your new site. Error: EPERM: operation not permitted, open 'D:\source\gitspace\orders\build\www\workbox-sw.prod.v2.1.2.js.map'
    at Promise.all.then.catch (D:\source\gitspace\orders\node_modules\workbox-build\build\lib\utils\copy-workbox-sw.js:17:13)
    at <anonymous>
error Command failed with exit code 1.
error Command failed with exit code 1.
error Command failed with exit code 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.


thanks!

When try to deploy a gateway application on heroku I get the same error.
...
[INFO] yarn run v1.3.2
[INFO] $ yarn run cleanup && yarn run webpack:prod:main && yarn run clean-www
[INFO] $ rimraf target/{aot,www}
[INFO] $ yarn run webpack -- --config webpack/webpack.prod.js --progress --profile
[ERROR] warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
[INFO] $ node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js --config webpack/webpack.prod.js --progress --profile
[ERROR] 35989ms building modules                                                                
[ERROR] 70ms sealing
[ERROR] 1ms optimizing 
[ERROR] 0ms basic module optimization 
[ERROR] 64ms module optimization
[ERROR] 0ms advanced module optimization 
[ERROR] 16ms basic chunk optimization       
[ERROR] 0ms chunk optimization 
[ERROR] 2ms advanced chunk optimization 
[ERROR] 2931ms building modules                                                                
[ERROR] 0ms module and chunk tree optimization 
[ERROR] 1ms chunk modules optimization 
[ERROR] 0ms advanced chunk modules optimization 
[ERROR] 9ms module reviving 
[ERROR] 0ms module order optimization 
[ERROR] 4ms module id optimization 
[ERROR] 4ms chunk reviving 
[ERROR] 0ms chunk order optimization 
[ERROR] 20ms chunk id optimization
[ERROR] 393ms hashing
[ERROR] 1ms module assets processing 
[ERROR] 463ms chunk assets processing
[ERROR] 6ms additional chunk assets processing 
[ERROR] 1ms recording 
[ERROR] 17178ms additional asset processing
[ERROR] 1ms chunk asset optimization 
[INFO] MergetJsonsWebpackPlugin compilation started...
[INFO] MergetJsonsWebpackPlugin compilation completed...
[ERROR] 504ms asset optimization
[INFO] MergetJsonsWebpackPlugin emit starts...
[INFO] MergetJsonsWebpackPlugin emit completed...
[INFO] MergetJsonsWebpackPlugin emit starts...
[INFO] MergetJsonsWebpackPlugin emit completed...
[INFO] MergetJsonsWebpackPlugin emit starts...
[INFO] MergetJsonsWebpackPlugin emit completed...
[ERROR]  95% emittingError: workbox-sw is needed by the service worker and could not be copied over to your new site. Error: ENOENT: no such file or directory, unlink '/Users/glauterlemos/Dropbox/DESENVOLVIMENTO/JAVA/indoorchannel/IDCGateway/target/www/workbox-sw.prod.v2.1.2.js'
[ERROR]     at Promise.all.then.catch (/Users/glauterlemos/Dropbox/DESENVOLVIMENTO/JAVA/indoorchannel/IDCGateway/node_modules/workbox-build/build/lib/utils/copy-workbox-sw.js:17:13)
[ERROR]     at <anonymous>
[ERROR] error Command failed with exit code 1.
[INFO] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[ERROR] error Command failed with exit code 1.
[INFO] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[ERROR] error Command failed with exit code 1.
[INFO] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:16 min
[INFO] Finished at: 2018-01-05T10:21:07-03:00
[INFO] Final Memory: 48M/422M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.6:yarn (webpack build prod) on project idc-gateway: Failed to run task: 'yarn run webpack:prod' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]

--- a/generators/client/templates/angular/_package.json
+++ b/generators/client/templates/angular/_package.json
@@ -127,7 +127,7 @@
     "webpack-notifier": "1.5.0",
     "webpack-visualizer-plugin": "0.1.11",
     "web-app-manifest-loader": "0.1.1",
-    "workbox-webpack-plugin": "2.1.2",
+    "workbox-webpack-plugin": "3.0.0-alpha.3",
     "write-file-webpack-plugin": "4.1.0"
   },
   "engines": {

--- a/generators/client/templates/angular/webpack/_webpack.prod.js
+++ b/generators/client/templates/angular/webpack/_webpack.prod.js
@@ -129,12 +129,6 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
             debug: false
         }),
         new WorkboxPlugin({
-          // to cache all under <%= BUILD_DIR %>www
-          globDirectory: utils.root('<%= BUILD_DIR %>www'),
-          // find these files and cache them
-          globPatterns: ['**/*.{html,bundle.js,css,png,svg,jpg,gif,json}'],
-          // create service worker at the <%= BUILD_DIR %>www
-          swDest: path.resolve(utils.root('<%= BUILD_DIR %>www'), 'sw.js'),
           clientsClaim: true,
           skipWaiting: true,
         })

